config/default.yml

```diff
@@ -1824,7 +1824,6 @@ Lint/MultipleComparison:
   Enabled: true
   VersionAdded: '0.47'
   VersionChanged: '1.1'
-  AllowMethodComparison: true
```

Comment:
@koic the cop caught this -- in #8953, this configuration appears to have been accidentally added to the configuration of `Lint/MultipleComparison` instead of `Style/MultipleComparison`!